data/json/mapgen/s_lightindustry_scen.json

```
@@ -212,7 +216,12 @@
         { "group": "elecsto_diy", "chance": 60, "repeat": 4, "x": [ 34, 39 ], "y": [ 12, 14 ] },
         { "group": "elecsto_diy", "chance": 60, "repeat": 4, "x": [ 34, 37 ], "y": [ 4, 5 ] }
       ],
-      "place_vehicles": [ { "vehicle": "cube_van", "x": 31, "y": 50, "rotation": 270 } ]
+      "vehicles": {
+        "Á": { "vehicle": "suburban_home_compact", "chance": 50, "rotation": 270 },
+        "É": { "vehicle": "industrial_transport_no_truck", "chance": 50, "rotation": 270 },
```

```suggestion
        "É": { "vehicle": "industrial_transport_no_truck", "rotation": 270 },
```
You should guarantee one vehicle for the scenario version of this location since the point of the scenario that uses it is to be able to start welding and modifying a vehicle from day one, it can happen that one spawns and sadly there is no vehicle to modify...